Tetris_formal.py

```diff
@@ -19,57 +23,97 @@
 
 
 ######### 基本資料
-screen_length = (
+SCREEN_LENGTH = (
     760  # 因為方塊的y值在左上角，所以方塊到最底層時時是800 - 40（方塊長度 = 40）
 )
-screen_width = 360  # 因為方塊的x值在左上角，所以方塊到最右方是400 - 40（方塊長度 = 40）
+SCREEN_WIDTH = 360  # 因為方塊的x值在左上角，所以方塊到最右方是400 - 40（方塊長度 = 40）
 
-block_width = 40
+BLOCK_WIDTH = 40
 
 
-######### 記錄各種方塊的上下左右
-class T_block():
-    
+class InitializeData:
+    """
+    各個方塊的初始條件
+    """
+
     def __init__(self, x, y):
         self.x = x
         self.y = y
-        
-    def get_direction(self, color):
-    
 
-        up = [[color, self.x, self.y], [color, self.x + 40, self.y], [color, self.x - 40, self.y], [color, self.x, self.y - 40]]
-        down = [[color, self.x, self.y], [color, self.x + 40, self.y], [color, self.x - 40, self.y], [color, self.x, self.y + 40]]
-        right = [[color, self.x, self.y], [color, self.x, self.y - 40], [color, self.x, self.y + 40], [color, self.x + 40, self.y]]
-        left = [[color, self.x, self.y], [color, self.x, self.y - 40], [color, self.x, self.y + 40], [color, self.x - 40, self.y]]
+    def color_or_picture(self, parameter, picture_root):
+        """
+        50%的機率選定會出現照片還是顏色，其中照片和顏色都是在其他方塊的class裡匯入的
+        """
+
+        preference = random.choice(["color", "picture"])
+
+        if preference == "color":
+            color = parameter
+
+        elif preference == "picture":
+            color = pygame.image.load(picture_root)
+            color = pygame.transform.scale(color, (40, 40))
+            color.blit(color, (self.x, self.y))
+
+        return color
+
+
+class TBlock(InitializeData):
+    """
+    上下左右的紀錄是為了之後旋轉時，能辨別它的上下左右的各個方塊的所在位置的座標是什麼。
+    """
+
+    def get_direction(self, color):
+        """
```

Comment:
Ditto. The description doesn't mean anything. Please briefly describe the function.